tw-tkms-starter/build.gradle

```diff
@@ -55,9 +55,10 @@ dependencies {
     testImplementation project(":tw-tkms-test-starter")
 
     testImplementation libraries.awaitility
-    if (!springBootVersion.startsWith("2.6")) {
-        testImplementation libraries.flywayMysql
+    if (springBootVersion.startsWith("3.3")) {
```

Comment:
That if might be better if we use `!springBootVersion.startsWith("3.2")` instead so that it will be forward compatible for new Spring Boot versions.